tests/python_test.py

```diff
@@ -29,15 +29,18 @@
 SECTOR_SIZE_512 = 512
 SECTOR_SIZE_4K = 4096
 
-
-@pytest.mark.parametrize("size,offset", [
-    # Smallest offset.
-    (LOCKSPACE_SIZE, 0),
-    # Large offset.
-    (LARGE_FILE_SIZE, LARGE_FILE_SIZE - LOCKSPACE_SIZE),
-])
-def test_write_lockspace(tmpdir, sanlock_daemon, size, offset):
-    path = str(tmpdir.join("lockspace"))
+TestParams=[
+        #size,offset,filename,encoding
+        (LOCKSPACE_SIZE, 0, u"ascii", None),
+        (LOCKSPACE_SIZE, 0, u"\u05d0", None),
+        (LARGE_FILE_SIZE, LARGE_FILE_SIZE - LOCKSPACE_SIZE, "ascii", None),
+        (LARGE_FILE_SIZE, LARGE_FILE_SIZE - LOCKSPACE_SIZE, u"\u05d0", "utf-8"),
+]
```

Comment:
This is faster, but the difference is very small, so it does not worth the mess of 
mixing unrelated tests.

To make the tests little cleaner and avoid repeating the parameters, we can have:
```
FILE_NAMES = [
    # name, encoding
    (u"ascii", None),
    (u"\u05d0", None),
    ("ascii", None),
    (u"\u05d0", "utf-8"),
]
```
The sizes should not be refactored like this since we test different values
(LOCKSPACE_SIZE, MIN_RES_SIZE). They happen to be the same for
512 bytes sector size, but this is little ugly. It also not required for adding the
file names and encoding tests.

And TestParams has too generic. We don't have any clue what kind of params are tested.